src/main/java/com/my/myspringdatabase/EmployeeTemplateDAO.java

```diff
@@ -0,0 +1,44 @@
+package com.my.myspringdatabase;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import javax.sql.DataSource;
+import java.util.List;
+
+public class EmployeeTemplateDAO implements EmployeeDao {
```

Comment:
create one more implementation of EmployeeDao by extending EmployeeTemplateDAO, please, then override Update method to be able to change name instead of age